README.md

```diff
@@ -1,4 +1,29 @@
-# smarthome-localhomesdk
+# Local Home SDK Virtual Environment
+
+A mock Local Home Platform implemented as stubs for the [Local Home SDK](https://github.com/actions-on-google/local-home-sdk)
+
+## Usage
+
+1. Place bundled Node Local Fulfillment app into `home-app` folder.
```

Comment:
Maybe we could also get rid of those instructions if this is packaged as a library.

Reply:
Will do in separate PR

Reply:
#21